town.logic

```diff
@@ -1,2 +1,46 @@
 area Town:
-  room Main
\ No newline at end of file
+  room Main:
+    node FigurineHouseDoorEast:
+      door FigurineHouseEast
+    node Main:
+      chest CafeLady
+      door Bakery
+      door East
+      door FigurineHouseWest
+      door HearthLedge
+      door HearthNorth
+      door HearthSOuth
+      door Julietta
+      door MayorEast
+      door MayorWest
+      door North
+      door Rem
+      door SchoolEast
+      door SchoolWest
+      door Simon
+      door South
+      door Stockwell
+      door Swiftblade
+      door Well
+      door West
+      entrance Crest
+      entrance TownDigging
+    Main <-> FigurineHouseDoorEast: item CarlovMedal
+    node MainBellHP:
+      chest MainBellHP
+    Main -> MainBellHP: item Cape 
+    node MainMinish
```

Comment:
This is missing a lot of doors (minish entrances/exits).

Reply:
Yeah, i just wanted to be sure what i had done was right so i wouldnt need to redo everything